docs/content/latest/benchmark/tpcc-ysql.md

```diff
@@ -43,90 +46,79 @@ $ cd tpcc
-   --execute &lt;arg&gt;           Execute the benchmark workload
--h,--help                          Print this help
-   --histograms                    Print txn histograms
-   --load &lt;arg&gt;              Load data using the benchmark's data loader
--o,--output &lt;arg&gt;            Output file (default System.out)
-   --runscript &lt;arg&gt;         Run an SQL script
--s,--sample &lt;arg&gt;            Sampling window
--v,--verbose                       Display Messages
+```sh
+$ ./tpccbenchmark --create=true --load=true
 ```
 
-Before starting the workload, you will need to load the data first.
+The above command runs the benchmark against the database running at 127.0.0.1 with 10 warehouses.
+To run the benchmark against a specific set of nodes with 100 warehouses:
 
 ```sh
-$ ./tpccbenchmark -c config/workload_all.xml --create=true --load=true
+$ ./tpccbenchmark --create=true --load=true \
+  --nodes=127.0.0.1,127.0.0.2,127.0.0.3 --warehouses=100 --loaderthreads=48
 ```
 
-Then, you can run the workload.
+You can then run the workload.
 
 ```sh
-$ ./tpccbenchmark -c config/workload_all.xml --execute=true -s 5 -o outputfile
+$ ./tpccbenchmark --execute=true -o outputfile
 ```
-
-You can also load and run the benchmark in a single step:
+or
 
 ```sh
-$ ./tpccbenchmark -c config/workload_all.xml --create=true --load=true --execute=true -s 5 -o outputfile
+$ ./tpccbenchmark --execute=true -o outputfile \
+  --nodes=127.0.0.1,127.0.0.2,127.0.0.3 --warehouses=100 -o outputfile
 ```
 
-The `config` directory has different configurations for various workloads. You can run any of those workloads by changing the configuration file.
+You can also load and run the benchmark in a single step:
 
 ```sh
-$ ./tpccbenchmark -c config/workload_1.xml --create=true --load=true --execute=true -s 5 -o outputfile
+$ ./tpccbenchmark --create=true --load=true --execute=true -o outputfile \
+  --nodes=127.0.0.1,127.0.0.2,127.0.0.3 --warehouses=100 -o outputfile
 ```
 
 ## Output
 
-The raw output is a listing of start time (in Java microseconds) and duration (microseconds) for each transaction type.
+Once the execution is done the TPM-C number along with the efficiency is printed.
 
 ```
-transaction type (index in config file), start time (microseconds),latency (microseconds)
-3,1323686190.045091,8677
-4,1323686190.050116,6800
-4,1323686190.055146,3221
-3,1323686190.060193,1459
-4,1323686190.065246,2476
-4,1323686190.070348,1834
-4,1323686190.075342,1904
+23:52:44,383 (DBWorkload.java:880) INFO  - Rate limited reqs/s: Results(nanoSeconds=500000707198, measuredRequests=23616) = 47.23193319534262 requests/sec
+23:52:44,383 (DBWorkload.java:885) INFO  - Num New Order transactions : 10376, time seconds: 500
+23:52:44,383 (DBWorkload.java:886) INFO  - TPM-C: 1245
+23:52:44,383 (DBWorkload.java:887) INFO  - Efficiency : 96.81181959564542
+23:52:44,384 (DBWorkload.java:722) INFO  - Output Raw data into file: results/output.5.csv
```

Comment:
Call this `results/outputfile.csv` for consistency with the statement above.

Reply:
Done.